utils/scriptHelper.js

```diff
@@ -11,17 +11,18 @@ const { loggers } = require('winston');
 const Logger = require("./Logger");
 const encrypt = require('./cryptHelper');
 const SystemMessageHelper = require("./SystemMessageHelper");
+const path = require('path');
 const isDevelopment = process.env.NODE_ENV == "development";
 
 class Script {
 	constructor() { }
 
 	async loadScript (scriptFolder, scriptName) {
 		try {
-			const props = require(`../scripts/${scriptFolder}/${scriptName}`);
+			const props = require(path.join(process.cwd(), `../scripts/${scriptFolder}/${scriptName}`));
 			if (props.cfg.enabled === true) {
-                const hash = encrypt.computeScriptSHA512(__dirname + `/../scripts/${scriptFolder}`);
-                const isValid = encrypt.checkScriptIntegrity(__dirname + `/../scripts/${scriptFolder}`, hash);
+                // const hash = encrypt.computeScriptSHA512(__dirname + `/../scripts/${scriptFolder}`);
```

Comment:
These are currently commended out due to no usecase, once finished encryption can be re-enabled.